src/test/rpc/Transaction_test.cpp

```diff
@@ -0,0 +1,282 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012-2017 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <test/jtx.h>
+#include <test/jtx/Env.h>
+#include <test/jtx/envconfig.h>
+#include <ripple/protocol/jss.h>
+#include <ripple/core/DatabaseCon.h>
+#include <ripple/protocol/ErrorCodes.h>
+
+namespace ripple {
+
+class Transaction_test : public beast::unit_test::suite
+{
+
+    void
+    testRangeRequest()
+    {
+        testcase("Test Range Request");
+
+        using namespace test::jtx;
+
+        const char* COMMAND   = jss::tx.c_str();
+        const char* BINARY    = jss::binary.c_str();
+        const char* NOT_FOUND = RPC::get_error_info(rpcTXN_NOT_FOUND).token;
+        const char* INVALID   = RPC::get_error_info(rpcINVALID_LGR_RANGE).token;
+        const char* EXCESSIVE = RPC::get_error_info(rpcEXCESSIVE_LGR_RANGE).token;
+
+        Env env(*this);
+        auto const alice = Account("alice");
+        env.fund(XRP(1000), alice);
+        env.close();
+        
```

Comment:
Extraneous white space.

Reply:
I thought breaking the statements at this point improves the readability and look of the code, but done anyway.

Reply:
@undertome I don't mind the newline at all - I'm in favor of keeping it. I'm flagging the eight spaces that start the line.

Reply:
Done.